### PR TITLE
feat: backstage with all postgres instances

### PR DESCRIPTION
**Pull request by eloycoto**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, use `fixes #<issue_number>(, fixes #<issue_number>, ...)` format, where issue_number might be a GitHub issue, or a Jira story (FLPATH-xxxx)*:
Fixes #

**Change type**
- [ ] New feature
- [ ] Bug fix
- [ ] Unit tests
- [ ] Integration tests
- [ ] CI
- [ ] Documentation
- [ ] Auto-generated SDK code

**Impacted services**
- [ ] Workflow Service
- [ ] Notification Service

**Checklist**
- [ ] Subject and description added to both, commit and PR.
- [ ] Relevant issues have been referenced.

### CODE REVIEW

File: hack/manifests/backstage/backstage.yaml [hunk 1]
```diff
@@ -26,7 +26,7 @@ spec:
             - "packages/backend"
             - "--config"
             - "/app/app-config.yaml"
-          image: quay.io/parodos-dev/backstage-parodos:latest
+          image: quay.io/parodos-dev/backstage-parodos:latest-openshift
```
Comment (pkliczewski) [inline hack/manifests/backstage/backstage.yaml:29]: Are there any changes how we build the image specific to ocp?

**Review by pkliczewski — COMMENTED** [review]
overall looks good, just one question

File: hack/manifests/backstage/backstage.yaml (hunk 1 above)
Comment (pkliczewski) [reply hack/manifests/backstage/backstage.yaml:29]: As agreed, let's have this in and check whether we can use the same image everywhere

**Review by pkliczewski — COMMENTED** [review]
/lgtm

**Review by RichardW98 — APPROVED** [review]

**Pull request merged**